zendframework/zendframework/ZF2015-06.yaml

```diff
@@ -0,0 +1,23 @@
+title:     XXE/XEE vector when using ZendXml on multibyte payloads
+link:      http://framework.zend.com/security/advisory/ZF2015-06
+cve:       CVE-2015-5161
+branches:
+    2.0.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.0.0,<2.0.99]
+    2.1.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.1.0,<2.1.99]
+    2.2.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.2.0,<2.2.99]
+    2.3.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.3.0,<2.3.8]
+    2.4.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.4.0,<2.4.6]
+    2.5.x:
+        time:     2015-08-03 15:13:58
+        versions: [>=2.5.0,<2.5.1]
```

Comment:
@stof While *one* use case for this repository is to provide machine-readable information that can be then consumed by automated tooling, another *stated purpose* in the readme is to allow *browsing existing vulnerabilities*.

Zend Framework provides distribution packages for ZF2 that pull in both the framework *and* ZendXml. For users of those download packages, if they have an earlier version *they are vulnerable*.

Please remember that *not everyone uses composer*. This is particularly true in enterprises (I hear it all the time particularly from enterprises using IBM i Series). As such, I'd argue that being more explicit here about where the vulnerabilities exist is better than not being explicit enough, and giving somebody a false sense of security.